platform_versioned_docs/version-24.1/seqerakit/commands.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: "Commands"
+description: "seqerakit command options"
+date: "21 Oct 2024"
+tags: [seqerakit, cli, automation, commands]
+---
+
+Use the `--help` or `-h` parameter to list available commands and options:
+
+```bash
+seqerakit --help
+```
+
+### Input
+
+Seqerakit supports input through paths to YAML configuration files or directly from standard input (stdin).
+
+- Using file path:
+
+    ```bash
+    seqerakit /path/to/file.yaml
+    ```
+
+- Using stdin:
+
+    ```console
+    $ cat file.yaml | seqerakit -
+    ```
```

Comment:
```suggestion
    ```bash
    seqerakit /path/to/file.yaml
    ```

- Using stdin:

    ```console
    $ cat file.yaml | seqerakit -
    ```
```
Minor comment.  The top example is path/to/file.yaml while the bottom is file.yaml. Unless there is a point to adding a complete or relative path, I would simplify. I'm also not sure if the $ is needed in the second example (I'm still getting my head around it). Similarly, what is the "-" at the end of the command? Is this required?

Reply:
Thanks. Made it consistent, and removed the $. Also split the commands from the output in separate code blocks elsewhere. As for the `-`, confirmed it is required:

![image](https://github.com/user-attachments/assets/694e5971-1512-4f1f-96c0-d6383d71a691)